### ISSUE TITLE
Using useDefaultTags + writePoint doubles escape characters

### ISSUE THREAD

**Issue by yourichabert** [influxdata/influxdb-client-js#391]
__Expected Behavior:__

When applying default tags with a call to `useDefaultTags` method of `WriteApi` interface, then writing a point to the database by calling the `writePoint` method, I would expect the ' ', '=', and all special characters contained in the tags values of the default tags to be escaped with a single backslash `\`, backslash that would then be removed by the database for the display.

__Current Behavior:__

Instead, both `useDefaultTags` and `writePoint` (which calls `toLineProtocol`) apply escaping to the special characters, so the backslashes are doubled, and a backslash appears in the display of the database.

__Possible Solution:__

Remove escaping of tags values from `useDefaultTags`.
Current code:

```
  useDefaultTags(tags: {[key: string]: string}): WriteApi {
    this.defaultTags = undefined
    Object.keys(tags).forEach((key: string) => {
      ;(this.defaultTags || (this.defaultTags = {}))[key] = escape.tag(
        tags[key]
      )
    })
    return this
  }
```
New code:
```
  useDefaultTags(tags: {[key: string]: string}): WriteApi {
    this.defaultTags = tags
  }
```
But I'm not aware of all the use cases and possible side-effects.

__Steps/Code to Reproduce the Problem:__

1. Code
```
import { InfluxDB, Point, WriteApi } from '@influxdata/influxdb-client';

const influxDB = new InfluxDB({ url: 'my-url', token: 'my-token' });
const writeApi = influxDB.getWriteApi('my-organization', 'my-bucket');
const defaultTags = { tagKey: "tag val=ue" };
writeApi.useDefaultTags(defaultTags);
const point = new Point();
writeApi.writePoint(point);
```
2. Look up in InfluxDB database

In the InfluxDB database, the value of the tag appears as `tag\ val\=ue` instead of `tag val=ue`.
In the project I work on, we are using Telegraf between the InfluxDB client and database, and when I print out the line protocol received by Telegraf in a file, I get `tag\\ val\\=ue`. This confirms that the reason why backslashes appear in the database it because they are sent doubled.

__Workaround:__

Storing the default tags in a dedicated variable, and applying them with `point.tag(tagName, tagValue)` after the call to the `Point` constructor.

__Specifications:__

- Version: latest, 1.10.0
- Platform: Windows 10

**Comment by sranka** [comment]
Thank you @yourichabert for using the client and reporting a descriptive issue. I am on it shortly.

**Comment by sranka** [comment]
The fix will be available in the nightly build once #392 is merged.
